platform-hub-web/src/app/projects/projects-form.html

```diff
@@ -49,6 +51,14 @@
             </textarea>
           </md-input-container>
 
+          <md-input-container class="md-block">
+            <label for="cost_centre_code">Cost centre code:</label>
+            <input
```

Comment:
Should this be coming from some dict of possible values to choose from I think - as optional field i.e. allowing no value? Probably some validation on inclusion in the model should follow too?